tee-worker/enclave-runtime/src/lib.rs

```diff
@@ -126,7 +126,10 @@ pub unsafe extern "C" fn init(
 	encoded_base_dir_size: u32,
 ) -> sgx_status_t {
 	// Initialize the logging environment in the enclave.
-	env_logger::init();
+	if_production_or!(
+		env_logger::Builder::new().filter(None, LevelFilter::Info).init(),
```

Comment:
@litentry/tee what do you think about forcing production build to use this log configuration ?

Reply:
Looks like it's related to #1514

Reply:
I'm fine with it - we'll have to keep an eye on INFO level logs tho to make sure there's no sensitive information